akka-stream/src/main/scala/akka/stream/javadsl/Queue.scala

```diff
@@ -32,8 +32,8 @@ trait SourceQueue[T] {
   def offer(elem: T): CompletionStage[QueueOfferResult]
 
   /**
-   * Method returns a [[CompletionStage]] that will be completed if the stream completes,
-   * or will be failed when the operator faces an internal failure.
+   * Method returns a [[CompletionStage]] that will be completed if this stage completes,
```

Comment:
I whink we are now using "operator" across the board so maybe here as well? (Also makes it less confusing with the completion stage)